install/installer.class.php

```diff
@@ -247,7 +247,12 @@ protected function createFirstAccess() {
          PluginFlyvemdmTaskstatus::$rightname      => READ,
       ];
 
-      $profileRight->updateProfileRights($_SESSION['glpiactiveprofile']['id'], $newRights);
+      $profiles_id = $_SESSION['glpiactiveprofile']['id'];
+      $profileRight->updateProfileRights($profiles_id, $newRights);
+      if($profiles_id != 4){
```

Comment:
don't hard code a ID of profile. Consider the variable $profiles_id is the ID of the super admin profile. Thus the if() {} is useless,